libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.scss

```diff
@@ -38,28 +68,49 @@
 }
 
 .sky-vertical-tabset-show-tabs-btn {
-  margin-top: $sky-margin;
+  margin-top: var(--sky-override-vertical-tabset-show-tabs-margin-top, 0);
 }
 
 .sky-vertical-tabset-group-container {
   flex-basis: 25%;
   overflow-y: auto;
+  padding: var(
+    --sky-override-vertical-tabset-group-container-padding,
+    var(--sky-space-inset-vertical-top-m) 0
+      var(--sky-space-inset-vertical-bottom-m)
+  );
 }
 
 :host-context(sky-modal-content) > :host .sky-vertical-tabset {
-  margin: -15px;
+  margin: var(
+    --sky-override-vertical-tabset-modal-margin-adjustment,
+    calc(var(--sky-space-inset-floated-top-m) * -1)
+      calc(var(--sky-space-inset-floated-right-m) * -1)
+      calc(var(--sky-space-inset-floated-bottom-m) * -1)
+      calc(
+        calc(
+            var(--sky-space-inset-pillarbox-2_3-left-m) +
+              var(--sky-space-inline-s)
+          ) * -1
+      )
+  );
+
+  &:has(sky-vertical-tabset-group) {
+    margin-left: var(
```

Comment:
The groups don't have the same inner padding and don't have the inset spacing. This adjusts for that so that we don't end up with group text to the left of the modal header and footer content.